app/src/main/java/org/squidfish/tuplenback/views/GameScreen.kt

```diff
@@ -91,8 +89,8 @@ fun GameScreen(
                 RepeatGuessButtons(
                     modifier = Modifier.fillMaxWidth().weight(1f),
                     recallGuesses = state.recallCheck,
-                    games = viewModel.game.modules,
-                    onGuess = { viewModel.onEvent(AppEvent.MakeMnemonicRepeatGuess(it)) },
+                    games = game.modules,
+                    onGuess = { onGuess(it) },
```

Comment:
`onGuess = onGuess`

Reply:
Inexplicably, ```onGuess``` is never called unless I use ```onGuess(it)```

Reply:
That is odd. Will have a look at it at some point